docs/requirements.md

```diff
@@ -7,19 +7,135 @@
+* Right below the map on the left should be the 3 frog sprite player lives.
+
+TODO: Add Game Mechanics like the dipping turtles, the obstacle/surface movement, and the ways to die.
+
+## Specific Game Mechanics
+### Death
+* Player can 'die' in the following ways:
+	** When player collides with a motor vehicle
+	** When player collides with water
+	** When player is on aset of turtles that 'dip' under the water
+
+### Dipping Turtles
+* Certain rows of turtles selected randomly should 'dip' under the water. When a row of turtles 'dip' under the water the turtles stop being a surface and if Frogger is on them when they dip, he collides with the water and 'dies'.
+
+### Obstacle and Surface Movement and Order
+* Obstacles and Surfaces move from one side of the map to the other in the road rows and the water rows.
+
+* The map should work and look exactly as described from the top row to the bottom:
+
+	** The Finishing Area/'Home'
+	** Medium Logs going from Left to Right
+	** A Row of 2 Turtles going from Right to Left
+	** Long Logs going from Right to Left
+	** Small Logs going from Right to Left
+	** A Row of 3 Turtles going from Right to Left
+	** Purple Grass/Safe Zone
+	** White Trucks going from Right to Left
+	** White Dune Buggies going from Left to Right
+	** Purple Cars going from Right to Left
+	** White Bulldozers going from Left To Right
+	** Yellow Taxis going from Right to Left
+	** Purple Grass/Safe Zone Where Frogger Spawns
+
+### Obstacle and Surface Speeds and Counts
+* Each obstacle and surface has a different speed ranging from fast, medium, and slow.
+	** The Yellow Taxis are slow
+	** The White Bulldozers are slow
+	** The Purple Cars are medium
+	** The White Dune Buggies are fast
+	** The White Trucks are fast
+	** The Row of 3 Turtles are slow
+	** The Row of  2 Turtles are fast
+	** The Small logs are slow
+	** The Long Logs are fast
+	** The Medium Logs are medium
+
+* Each obstacle and surface have a different maximum and minimum that are allowed on the screen at a time.
+	** All motor vehicles should have a minimum of 2 vehicles per vehicle row and a maximum of 3.
+	** Short logs should have a minimum of 2 short logs per short log row and a maximum of 3.
+	** Medium logs should have a minimum of 3 medium logs per medium log row and a maximum of 3 and a half.
+	** Long logs should have a minimum of 2 per long log row and a maximum of 2 and a half
+	** The row of 3 turtles should have a minimum of 3 in that turtle row and maximum of 3 and a half.
+	** The row of 2 turtles should have a minimum of 3 in that turtle row and a maximum of 4.
+
+
+### The Safe Zones
+* The purple grass safe zones are rows that don't have any obstacles or surfaces. It is just a row where Frogger can stop without fear of 'drowning' or becoming 'roadkill'.
+
+
+
+TODO: This point and lower need to be revised.
+## Player Characteristics
```

Comment:
This has basic instructions needed to play, like noting that arrow keys are needed in order to move, it may be better above the description of the game board?

Reply:
I don't think it needs to be in the game board description but maybe in game mechanics? I'll take a look.